.azuredevops/pipelineTemplates/jobs.validateModulePester.yml

```diff
@@ -44,22 +44,22 @@
 
 parameters:
   # Pipeline-related parameters
-  serviceConnection: '$(serviceConnection)'
-  poolName: '$(poolName)'
-  vmImage: '$(vmImage)'
+  serviceConnection: "$(serviceConnection)"
```

Comment:
Please search and replace `"` -> `'`. There is a setting to do this in VSCode `"yaml.format.singleQuote": true` which we have in the .vscode/settings.json file.

Reply:
```suggestion
  serviceConnection: '$(serviceConnection)'
```